### ISSUE TITLE
Bug: Failed to initialise Minecraft path ``

### ISSUE THREAD

**Issue by Lamparter** [RiversideValley/Emerald#48]
### ⏱️ Before you start...

- [X] Are you on the latest version of Emerald? You might be using an old version.
- [X] Have you checked whether or not a similar issue has already been reported?

### 🔢 What version of Emerald are you on?

Unavailable due to app inaccessibility due to this error.

### 📄 Description

On launch, Emerald displays the following message box:
```
ERROR

Failed to initialise Minecraft path at ``
```

This error occurs when Minecraft is installed via the Xbox app/Microsoft Store

### 🤔 Expected behavior

App should not display this error; default installation location registered

### 📸 Assets

![image](https://github.com/user-attachments/assets/5fe02ec9-2481-49cb-ba91-7a04360a0795) [`Developer Sanctuary > #riverside`](https://discordapp.com/channels/714581497222398064/1093841254611157122/1277282841088233472)

**Comment by Lamparter** [comment]
I'm really confused why this happened.

**Comment by jarno9981** [comment]
maybe endpoint path or lets say like hardcoded %appdata% as path or something just a idea

**Comment by Lamparter** [comment]
This comes from a JSON serialisation exception
Though adding a fallback default might be a good idea.

**Comment by NoobNotFound** [comment]
> This comes from a JSON serialisation exception. Though adding a fallback default might be a good idea.

yeah but I thought instead of just falling back, notifiying the error to the user could be useful. then they can take an action (retry or change path). but yeah we can implement it in the uno rewrite.
